formhero/providers/dbform/tests.py

```diff
@@ -1,3 +1,33 @@
 from django.test import TestCase
 
-# Create your tests here.
+from formhero.apps.models import App
+from formhero.forms.models import Form
+from formhero.providers.dbform import Backend
+from formhero.providers.dbform.models import FormEntry
+
+
+class DatabaseFormTest(TestCase):
+
+    def setUp(self):
+        self.an_app = App.objects.create(name='this_app')
+        self.data = {'name': 'Dave',
+                'inquirer email': 'dave@example.com',
+                'message': 'This is a test message'
+                }
+        self.a_form = Form.objects.create(app=self.an_app, name='Form name', handler='db', config=self.data)
```

Comment:
config is specific configuration that a handler might use such as SENDGRID_API_KEY, or EMAIL_HOST